bears/vcs/git/GitCommitBear.py

```diff
@@ -236,6 +239,8 @@ def check_body(self, body,
                                     expressions in this list will be ignored.
         :param body_regex:          If provided, checks the presence of regex
                                     in the commit body.
+        :param verify_email:         Checks validity of emails in commit body if
+                                    present.
```

Comment:
Line is longer than allowed. (80 > 79)

*LineLengthBear, severity NORMAL, section `linelength`.*